[PL-39070] [DOC-3313] Update SAML provider field restrictions for multiple IDP (#3400)
harness/developer-hub: docs/platform/authentication/multiple-identity-providers.md

    
    b. If one or more SAML providers are configured, select **Add SAML Provider**.
   The SAML Provider settings appear.

> 3. In **Name**, enter a name for the SAML provider. Names are restricted to alphanumeric characters, `_`, `-`, `.`, and spaces.

Author: ```suggestion
3. In **Name**, enter a name for the SAML provider. Names can only contain alphanumeric characters, `_`, `-`, `.`, and spaces.
```
